Beams/Logic/vtkSlicerBeamsModuleLogic.cxx

```diff
@@ -397,7 +397,7 @@ void vtkSlicerBeamsModuleLogic::UpdateIECTransformsFromBeam(vtkMRMLRTBeamNode* b
   // Set beam angles to IEC logic
   this->IECLogic->UpdateGantryToFixedReferenceTransform(beamNode->GetGantryAngle());
   this->IECLogic->UpdateCollimatorToGantryTransform(beamNode->GetCollimatorAngle());
-  this->IECLogic->UpdatePatientSupportRotationToFixedReferenceTransform(beamNode->GetCouchAngle());
+  this->IECLogic->UpdatePatientSupportRotationToFixedReferenceTransform(-1. * beamNode->GetCouchAngle());

```

Comment:
If signs are flipped here, then the comment

```
  // Set the inverse of the couch angle as now what moves is the room (treatment machine) around the patient,
  // so the patient support table top (couch) always stays stationary in RAS.
```

should be moved from the other files to this one.

Reply:
Agreed.

Reply:
Fixed.